src/helper/class-ee-site.php

```diff
@@ -223,6 +223,7 @@ protected function delete_site( $level, $site_url, $site_fs_path, $db_data = []
 			}
 		}
 
+		\EE::do_hook( 'site_cleanup', $site_url );
```

Comment:
@mrrobot47 Please add some comment above this hook. Like: What it can be use for and when and what's $site_url is use for.

Reply:
Added. Please check of this much info is sufficient.